packages/features/ee/workflows/lib/constants.ts

```diff
@@ -11,7 +11,7 @@ export const WORKFLOW_TRIGGER_EVENTS = [
 export const WORKFLOW_ACTIONS = [
   WorkflowActions.EMAIL_HOST,
   WorkflowActions.EMAIL_ATTENDEE,
-  WorkflowActions.EMAIL_ADDRESS,
+  //WorkflowActions.EMAIL_ADDRESS, Disabling for now due to abuse episode
```

Comment:
```suggestion
```

Reply:
not sure if // works. maybe just delete it?

Reply:
It does work. But I agree that removing might make more sense. Adding it back later is easy.